ietf/templates/doc/opengraph.html

```diff
@@ -0,0 +1,16 @@
+{# Copyright The IETF Trust 2016-2020, All Rights Reserved #}
+{% load origin %}
+{% load static %}
+{% origin %}
+<meta property="og:title" content="{{ doc.title }} ({% if doc.get_state_slug == 'rfc' and not snapshot %}RFC {{ rfc_number }}{% else %}{{ name }}-{{ doc.rev }}{% endif %})">
+<meta property="og:url" content="{{ settings.IDTRACKER_BASE_URL }}{{ request.path }}">
+<meta property="og:image" content="{{ settings.IDTRACKER_BASE_URL }}{% static 'ietf/images/ietf-logo-nor-512.png' %}">
+<meta property="og:image:alt" content="Logo of the IETF">
+<meta property="og:image:width" content="512">
+<meta property="og:image:height" content="512">
+<meta property="og:type" content="article">
+<meta property="og:site_name" content="IETF Datatracker">
+<meta property="og:article:section" content="IETF - Internet Engineering Task Force">
+<meta property="og:description" content="{% for author in doc.documentauthor_set.all %}{{ author.person.name}}{% if not forloop.last %} and {% endif %}{% endfor %}. {{ doc.title }}. {% if doc.get_state_slug == 'rfc' and not snapshot %}RFC {{ rfc_number }}, {{ published.time|date:'M Y' }}, {{doc.pages}} pages{% else %}{{ name }}-{{ doc.rev }}; work in progress{% endif %}.">
```

Comment:
This is noisy. Social platforms (e.g., [twitter](https://cards-dev.twitter.com/validator)) have metadata previewers that let you see how it will be presented.

Can you take the same approach that [I took](https://github.com/ietf-tools/xml2rfc/pull/777)?

Reply:
Done, please take a look?